index.html

```diff
@@ -905,8 +905,8 @@ <h2 class="fancy-title text-center color-white"><span>Our Customers & Testimonia
 	<script type="text/javascript" src="./methodology/methodology.json"></script>
 	<script type="text/javascript" src="./methodology/methodology.js"></script>
 
-	<script type="text/javascript" src="./faq/FAQ.json"></script>
-	<script type="text/javascript" src="./faq/Faq.js"></script>
+	<script type="text/javascript" src="./faq/faq.json"></script>
+	<script type="text/javascript" src="./faq/faq.js"></script>

```

Comment:
file name that exists is  faq/Faq.js but calling other name. Same with json file